cves/CVE-2018-13304.yml

```diff
@@ -331,11 +341,11 @@ lessons:
     applies:
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: This vulnerability was caused by too many types of checks on what the profile should be. 
```

Comment:
Too many checks? Or was this an issue of each check being required, but just not implemented well?

Reply:
A little bit of both, I re-wrote it so it should be more clear